### PR TITLE
Add/wallpaper

### PR DESCRIPTION
**Pull request by SorataBaka**
Added new wallpaper command and **fixed issue with help embed replying every time someone sends a message.**

Command Added: 

- cuy/wp (list / {genre})
- cuy/wallpaper (list / {genre})

Help Embed Updated: 
image

Integrated with "CuyAPI" located in the following repository: 
https://github.com/cuybot-company/CuyAPI

In reference to the following issue: 
https://github.com/cuybot-company/cuybot-discord/issues/34

### CODE REVIEW

File: response/help.py [hunk 1]
```diff
@@ -8,20 +8,21 @@ def __init__(self, user_message, bot_send):
...
-    # untuk sementara hard code semua command
```
Comment (SorataBaka) [inline response/help.py:11]: Embed section was not indented which caused it not to be included in the if statement

File: response/help.py [hunk 2]
```diff
@@ -8,20 +8,21 @@ def __init__(self, user_message, bot_send):
...
-    embed.add_field(name="Tebak usia", value="usia *nama*", inline=True)
-    embed.add_field(name="Informasi spesifikasi smartphone", value="hp *merk_hp*, handphone, mobile, phone", inline=True)
-    embed.add_field(name="Informasi daftar smartphone terbaru", value="hp baru, handphone baru, mobile baru, phone baru", inline=True)
-    embed.add_field(name="Cari lirik lagu:", value="lirik, lyric, lyrics", inline=True)
-    embed.add_field(name="Mobile Legends Redeem", value="ml, mlredeem", inline=True)
-    embed.add_field(name="Reputation", value="rep", inline=True)
-    embed.set_footer(text="Bot masih dalam tahap pengembangan.")
-    await self.bot_send(embed=embed)
+      # untuk sementara hard code semua command
+      embed=discord.Embed(title="Cuybot Help", description="Command dasar pemanggilan bot cuy/(command) tanpa tanda kurung", color=0x50d396)
```
Comment (SorataBaka) [inline response/help.py:12]: Indentation fixed this issue by merging it with the if statement

**Comment by afrizaldea** [comment]
please fix this little conflict @SorataBaka thank u :)

File: response/help.py (hunk 2 above)
Comment (Rizsyad) [reply response/help.py:12]: please using helper embeed, you can see example in response/help.py

Comment (SorataBaka) [reply response/help.py:12]: > please using helper embeed, you can see example in response/help.py

what do you mean?

**Pull request closed without merging**